Source/missiles.cpp

```diff
@@ -3818,39 +3818,18 @@ void MI_FireRing(int i)
 		if (nMissileTable[dp] || missile[i]._miVar8)
 			missile[i]._miVar8 = 1;
 		else
-			AddMissile({ tx, ty }, { tx, ty }, 0, MIS_FIREWALL, TARGET_BOTH, src, dmg, missile[i]._mispllvl);
+			AddMissile({ tx, ty }, { tx, ty }, 0, type, TARGET_BOTH, src, dmg, missile[i]._mispllvl);
 	}
 }
 
-void MI_LightningRing(int i)
+void MI_FireRing(int i)

```

Comment:
It would have been best if you had made this change while the two functions where still identical.

Reply:
I'll just squash and force push.

Reply:
acceptable